roles/reset/tasks/main.yml

```diff
@@ -54,3 +54,6 @@
   file:
     path: /tmp/k3s
     state: absent
+
+- name: Remove LXC modifications if applicable
+  include_tasks: lxc.yml
```

Comment:
Shouldn't we only include this task list if `proxmox_lxc_configure` is true?

Reply:
I thought about that too, but even if it is not true it still guarantees we didn't leave any trace of this behind. We can just add a when to check for it though if we think we should limit the scope.

Reply:
ah fair actually, if the block isn't there this should just do nothing yeah? probably fine to always run it

Reply:
> ah fair actually, if the block isn't there this should just do nothing yeah? probably fine to always run it

Yeah it will check for this managed block and just make sure it is absent from the file. So if this block was never hit in the deploy, it will never be there after wards. It also will not attempt to create the file on the reset either. 

I was more looking at covering all the bases incase someone used a variable they did not mean to and change it between the reset run.

Reply:
yeah, gotcha, think this is fine as is, good point